tests/security/ibmtss/ibmtss_basic_function.pm

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Update IBM's Trusted Computing Group Software Stack (TSS) to the latest version.
+#          IBM has tested x86_64, s390x and ppc64le, we only need cover aarch64
+#          This test module covers basic function test
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#101088, poo#102792, tc#1769800
+
+use base 'opensusebasetest';
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+use version_utils 'is_sle';
+use registration 'add_suseconnect_product';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Install required develop packages
+    if (is_sle('>=15')) {
```

Comment:
will this be **15-sp4** ?

Reply:
Good comments, I planed to do so, however, this should be supported from sles15, I just copied from others' code :)
It is safe to keep in sle15, right?

Reply:
yes, I think so, I am not sure if it is the specific feature for 14-SP4 only, but you confirm that can support in SLE15 all SPx, it is find to keep it. :)